index.js

```diff
@@ -1207,6 +1207,10 @@ ArrayTome.prototype.shift = function () {
 		diff(this, 'shift');
 	}
 
+	if (ObjectTome.isObjectTome(out)) {
```

Comment:
Why are you only doing this for ObjectTomes?

The interface should be consistent, always return tomes or never return tomes.

I think you'd be better off just doing `return Tome.conjure(out)`